src/components/embedded/EmbeddedEvent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import api from '../../api';
+import { EventAction } from '../../models/EventAction';
+import EventBodyCard from '../event/EventBodyCard';
+import SplashScreen from '../SplashScreen';
+
+function EmbeddedEvent({ eventId }: { eventId: string }) {
+	const [event, setEvent] = useState<EventAction | null>(null);
+
+	useEffect(() => {
+		getMessage();
+	}, []);
+
+	async function getMessage() {
```

Comment:
function getEvent()

there is a problem here that on error component is stuck on showing loading.
part of the problem is that api.events.getEvent return null if event hasnt been found.
i think we made api return null if there is an error because we handled all errors inside fetch-intercept.ts.

What we can do is to call `fetch(/th2-commonv3/backend/event/${id}?${params})` in this component as well as inside EmbeddedMessage. then handle error and throw it.

we can take this logic from fetch-intercept which returns error message based on status and make another function getErrorMessage to use it here, in EmbeddedMessage component, and in fetch-intercept.ts